### ISSUE TITLE
Issue with SDF while reading the .sdf files using python

### ISSUE THREAD

**Issue by Khatua03** [Warwick-Plasma/epoch#438]
Hii All. I tried to read the sdf files using python for which I built the sdf_helper. After that, I am getting an error message "Your sdf python module is too old for this version of sdf_helper. Either upgrade to sdf python 2.2.0 or newer, or downgrade sdf_helper". Can anyone suggest to me how to resolve this?

I tried to upgrade sdf, but couldn't find sdf 2.2.0.

**Comment by Status-Mirror** [comment]
Hi @Khatua03,

Can I check if you were installing the Python packages according to these instructions: https://epochpic.github.io/documentation/visualising_output/python.html

Cheers,
Stuart

**Comment by Khatua03** [comment]
yes, I have installed all the packages and abled to run "make sdfutils" and I guess sdf_helper is created. Then I tried to run the python code for reading .sdf files. For reference, I am attaching the screenshots.

Thank you.
DP Khatua.

![image](https://user-images.githubusercontent.com/63596383/215854464-6b4717ff-797e-42de-89d9-17af33d4c7e6.png)
![image](https://user-images.githubusercontent.com/63596383/215855822-5310aa24-578a-4d92-b2fc-cc85adde1a92.png)
